BLSQ/iaso
# IA-3466 Implement permissions for users with org unit types restrictions

Implement additional permissions for users with org unit types restrictions.

Related JIRA tickets : [IA-3466](https://bluesquare.atlassian.net/browse/IA-3466)

## Changes

- implement additional permissions for users with org unit types restrictions:

    - in bulk create users (via CSV)
    - in `api/profiles` update and create
    - in profile bulk update

- add validation in `api/profiles` **create** the same way it's implemented in **update**
    - also move methods inside the class so they are logically grouped

## How to test

Use the UI or check the unit tests


[IA-3466]: https://bluesquare.atlassian.net/browse/IA-3466?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
